main.go

```diff
@@ -40,7 +42,7 @@ func transform(in io.Reader, out io.Writer, C int, B bool) error {
 
 	var result string
 	if B {
-		result = b64.URLEncoding.EncodeToString(f)
+		result = b64.StdEncoding.EncodeToString(f)
```

Comment:
let's get rid of this function and rewrite logic like:

```go
type Transformer interface {
    Transform(in io.Reader) (string, error)
}

type CaesarTransformer sturct {
    Shift int
}

func NewCaesarTransformer(shift int) *CaesarTransformer {
    return &CaesarTransformer{shift: shift}
}

func (t *CaesarTransformer) Transform (r io.Reader) (string, error) {
     b, err := io.ReadAll(r)
     if err != nil ....
     
     // make transformation using t.shift

    return transformedString, nil
}

// same for other transformers
```

Then in handler you already know which transform to use:

```go
// caesar handler 

shiftParam := r.URL.Query().Get("shift")
shift, err := strconv.Atoi(shiftStr)
// handle err

res, err := NewCaesarTransformer(shift).Transform(r.Body)
// handle err

w.Write([]byte(res))
```

same for other handlers


When working with flags, decide which transformer to create:

```go
var transformer Transformer

if caesar {
    transformer = NewCaesarTransformer(shift)
}
...

res, err := transformer.Transform()
// handle err
```

then write the result to file, stdout, etc.